src/components/calcite-icon/calcite-icon.tsx

```diff
@@ -37,6 +37,14 @@ export class CalciteIcon {
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * Custom height (overrides the scale property)
```

Comment:
What prompted these? Seems like you could just set the width of the `calcite-icon` component itsself?

Reply:
Yeah I feel like we shouldn't make this as accessible - you can always override within another component or externally with css scale? Since each `scale` here actually renders a different svg with different details I think we want to preserve that as much as possible...

Reply:
Yeah, I wanted to understand this decision to render different SVGs for each scale better.  Why not just have a single SVG and render it at any size needed?  In the past this is what I've done with SVG icons.

Also, this is what it looks like when styling `calcite-icon` directly with a custom height and width:
![image](https://user-images.githubusercontent.com/821864/88306646-d73adb80-ccbf-11ea-982b-fb2ad613d4c2.png)

The SVG itself needs to change in size, and setting the style this way only affects the host element.

Reply:
Also, I asked this question in the Calcite Components channel, but never got any response: https://teams.microsoft.com/l/message/19:fd15b51dacd24e70895ec1218a54ae06@thread.skype/1594683185270?tenantId=aee6e3c9-711e-4c7c-bd27-04f2307db20d&groupId=56fae21a-9407-4943-859f-a9bfcf0bbad3&parentMessageId=1594683185270&teamName=Calcite%20Design%20System&channelName=Calcite%20Components&createdTime=1594683185270

Reply:
OK, digging in on the branch locally, I see what you mean about only affecting the host. We could solve this by giving the SVG `width:100%; height: 100%;` and then setting the width and height on the host, maybe? Or perhaps just an xl size?

Reply:
Would prefer the former option between those two vs. a new scale modifier 🤷‍♂️

Reply:
Ok!  I finally got @paulcpederson 's idea to use 100% width and height to work.  Give a tug on the branch and see for yourself!